src/components/SupportalActionModal.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import useLocalize from '@hooks/useLocalize';
+import ConfirmModal from './ConfirmModal';
+
+type SupportalActionModalProps = {
+    isSupportalActionRestrictedModalOpen: boolean;
+    hideSupportalModal: () => void;
+};
+
+function SupportalActionModal({isSupportalActionRestrictedModalOpen, hideSupportalModal}: SupportalActionModalProps) {
```

Comment:
What do y'all think about shortening `isSupportalActionRestrictedModalOpen` to just `isOpen` since that's a pretty clear prop name internal to this component? 🤔

Reply:
Hmmm what about `isModalOpen`? 😅  I just don't think we need to be TOO specific, but I do agree `isOpen` is a bit too generic 😅